File: app/src/main/java/com/tianchuang/ihome_b/mvp/BaseRequestView.java
package com.tianchuang.ihome_b.mvp;

/**
 * Created by Abyss on 2017/4/1.
 * description:
 */

public interface BaseRequestView extends BaseView {
    void showToast(String message);

    void showErrorPage();

    void showSucceedPage();

    void showEmptyPage();

    void showProgress();

    void dismissProgress();

}

File: app/src/main/java/com/tianchuang/ihome_b/mvp/BaseRequestView.kt
package com.tianchuang.ihome_b.mvp

/**
 * Created by Abyss on 2017/4/1.
 * description:
 */

interface BaseRequestView : BaseView {
    fun showToast(message: String)

    fun showErrorPage()

    fun showSucceedPage()

    fun showEmptyPage()

    fun showProgress()

    fun dismissProgress()

}
